src/nlohmann/json-schema.hpp

```diff
@@ -128,7 +128,7 @@ class JSON_SCHEMA_VALIDATOR_API json_uri
 namespace json_schema
 {
 
-extern json draft7_schema_builtin;
+extern json draft_2020_12_schema_builtin;
```

Comment:
I left this general approach here, because I suspect that the intent is to allow for multiple versions of the schema to be included simultaneously -- and validate against the one indicated by `$schema`. Is that correct?
If not, then this variable should be renamed to something more generic, like `schema_builtin`.